src/modules/Verifications/components/Status/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
if it's like our status component in General, can  you use it?

Reply:
Plaese let's use this. I handled verification status in this component